messaging/src/main/java/org/axonframework/commandhandling/SimpleCommandBus.java

```diff
@@ -334,6 +353,17 @@ public Builder defaultCommandCallback(@Nonnull CommandCallback<Object, Object> d
             return this;
         }
 
+        /**
+         * Sets the {@link SpanFactory} implementation to use for providing tracing capabilities.
```

Comment:
Builder methods their JavaDoc generally state the default setting in absence of invocation if there's a default setting.

Furthermore, I would expect the provided field to be asserted to be not `null` so that the user's protected during start-up.